src/cli/node/json_rpc_client.rs

```diff
@@ -719,7 +719,10 @@ pub fn data_request_report(
                         let honest = match dr_info.tally.as_ref() {
                             None => format!(""),
                             Some(tally) => {
-                                if tally.slashed_witnesses.contains(&pkh) {
+                                // TODO: Need review by tmpolaczyk
```

Comment:
@tmpolaczyk could you review that part?

Reply:
This condition should be "if the witness was not rewarded". But now there are 3 possible outcomes, so we should re-think the `dataRequestReport` method.

The current behavior is:
* Honest: "+reward"
* Liar: red text "-0"

I propose:

* Honest: "+reward"
* Liar: red text "-0"
* Error: red text "+reward"

This way will be relatively simple to implement but it may be difficult for users to understand the difference, unless we explain all the consensus logic in `dataRequestReport --help`.

So yeah, we should add an additional case to this if. Will see if I can write a commit later.

Reply:
we consider an error if it is out of consensus and an error, so it should not have a reward. And I think we have to include -collateral for the liars?

Reply:
An error in consensus should be considered as honest

Reply:
> And I think we have to include -collateral for the liars?

Currently, `dataRequestReport` never shows `-collateral` when you lose the collateral, but that would make sense.

Reply:
Proposal number 2:
* Honest: "+reward"
* Liar: red text "-collateral"
* Error: "+0"

Reply:
Proposal number 3:
* Honest: "+reward" "+rep"
* Liar: red text "-collateral"
* Error: "+reward"

Although I'm not sure if it is possible to get the amount of reputation that has been rewarded to witnesses.

Reply:
applied!!